### PR TITLE
Basic Level Project AngularJS - QuickNotes 

### PR DESCRIPTION
**Pull request by osmoosiis**
### Issue Title: QuickNotes using AngularJS.

- **Name of the project** : QuickNotes
- **Category:** AngularJS
- **Name:** Ishavdeep Kaur 
- **GitHub ID:** @osmoosiis
- **Idenitfy yourself:** GSSOC Contributor 

Closes: #39

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots [📷]
Screenshot (393)

### CODE REVIEW

File: Angular-JS-Projects/Basic/Quick-Notes/Screenshot (392).webp
Comment (Avdhesh-Varshney) [inline Angular-JS-Projects/Basic/Quick-Notes/Screenshot (392).webp:1]: change the name to `screenshot.webp`

File: Angular-JS-Projects/Basic/Quick-Notes/5470877.webp
Comment (Avdhesh-Varshney) [inline Angular-JS-Projects/Basic/Quick-Notes/5470877.webp:1]: Change the file name with `bg.webp` file and don't forget to update the bg file in styles.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Correct these changes.

**Comment by osmoosiis** [comment]
@Avdhesh-Varshney added the changes .

**Review by Avdhesh-Varshney — APPROVED** [review]
@osmoosiis PR Approved. 🎉
But don't commit in any un-necessary files as done in README file of Next-JS-Projects.

**Comment by osmoosiis** [comment]
> @osmoosiis PR Approved. 🎉
> But don't commit in any un-necessary files as done in README file of Next-JS-Projects.

Yeah sorry about that i didn't realise I was on next js readme file .

**Pull request merged**